examples/tokens/token_create_transaction_token_fee_schedule_key.py

```
@@ -126,31 +143,45 @@ def attempt_fee_update(client, token_id, fee_schedule_key, description):
         # Query final fees
-        query_token_fees(client, token_with_key, "Token with fee_schedule_key (after update)")
-        query_token_fees(client, token_without_key, "Token without fee_schedule_key (after update)")
-        
+        query_token_fees(
+            client, token_with_key, "Token with fee_schedule_key (after update)"
+        )
+        query_token_fees(
+            client, token_without_key, "Token without fee_schedule_key (after update)"
+        )
+
     except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)